src/schema/mutation/editApplication.ts

```diff
@@ -27,6 +48,7 @@ export default {
     pages: { type: new GraphQLList(GraphQLID) },
     settings: { type: GraphQLJSON },
     permissions: { type: GraphQLJSON },
+    templates: { type: GraphQLJSON },
```

Comment:
@matheus-relief instead of doing that, I think we should add migrations:
- addTemplate ( args: application / subject / type / body )
- editTemplate ( args: id / subject / body )
- deleteTemplate ( args: id )